blue_hardware:
    joint_names: ['base_roll_joint']
    motor_names: ['base_roll_motor']
    motor_ids: [1]
    gear_ratios: [7.1875] # 8.24448529412 results from 7.1875 * (39 / 34) from the output gear
    current_to_torque_ratios: [0.626] # 1.0 is decent approx for small motors
    current_offset: [0.0]
    calibration_initial_angles: [0.0]
    hardstop_start_angles: [0.0] # angles that you the robot is in for hardstop calibration
    joint_torque_directions: [1.0]
    motor_current_limits: [0.2]
    id_torque_gains: [0.0]
    softstop_min_angles: [-8.2]
    softstop_max_angles: [8.2]
    differential_pairs: []
    #vive or rviz
    target_mode: "vive"
    softstop_torque_limit: 0.0
    softstop_tolerance: 0.0
    accelerometer_calibration: false
    endlink: "shoulder_link"
